src/frontend/components/ProductCard/ProductCard.jsx

```diff
@@ -0,0 +1,76 @@
+import "./ProductCard.css";
+const ProductCard = ({
+  discountPrice,
+  originalPrice,
+  img,
+  name,
+  tag,
+  savePercent,
+  rating,
+}) => {
+  const isDiscountGiven = savePercent === 0 ? false : true;
+  return (
+    <div className="card-wrapper product-card place-center p-relative">
+      {tag === "" ? null : (
+        <span className="vertical-card-badge text-sm p-3 p-absolute">
+          {tag}
+        </span>
+      )}
+      <div className="px-4">
+        <img
+          className="img-responsive d-block m-auto product-card-img"
+          src={img}
+          alt="headphone"
+        />
+      </div>
+      <div className="p-2">
+        <p className="font-wt-bold text-center text-sm">{name}</p>
+      </div>
+      <div className="px-8 py-2">
+        {isDiscountGiven ? (
+          <div className="product-price-wrapper">
+            <span className="product-discount-price text-sm font-wt-bold">
+              {`Rs. ${discountPrice}`}
+            </span>
+            <span className="product-original-price text-sm ml-6">{`Rs. ${originalPrice}`}</span>
+          </div>
+        ) : (
+          <div className="product-price-wrapper">
+            <span className="product-discount-price text-sm font-wt-bold">
+              {`Rs. ${discountPrice}`}
+            </span>{" "}
```

Comment:
this can be shortened. isDiscountGiven can be used to show/hide just the `<span></span>`